### DISCUSSION TITLE
Need help building a zombie game with automatic team conversion on death

### DISCUSSION THREAD

**Discussion by mcdjhorton** [Pryaxis/TShock#1720] [General]
I'm making a zombie game and I need help making so everyone starts on blue team and if you die you get transferred to green, Pvp is on automatically and if you try to go back to blue you get sent back to green.

**Comment by hakusaro** [comment]
I changed the title of this discussion on your behalf to be more specific

**Comment by moisterrific** [comment]
You can try seeing how existing PvP mini game plugins work and use parts as the basis for your custom plugin idea. 

This is a good example to get started from I think: https://github.com/Interverse/BatteriesNotIncluded
